### PR TITLE
feat: add 2016S_FE_AM_1

### PR DESCRIPTION
**Pull request by kulaizki**

### CODE REVIEW

**Review by Luzefiru — APPROVED** [review]

**Review by Luzefiru — APPROVED** [review]

File: .obsidian/plugins/obsidian-spaced-repetition/data.json
Comment (sethonne) [inline .obsidian/plugins/obsidian-spaced-repetition/data.json:1]: Why was this changed?

**Review by sethonne — CHANGES_REQUESTED** [review]

File: .obsidian/plugins/obsidian-spaced-repetition/data.json
Comment (kulaizki) [reply .obsidian/plugins/obsidian-spaced-repetition/data.json:1]: cards on my phone have an inch for its width with the previous setting T.T

Comment (kulaizki) [reply .obsidian/plugins/obsidian-spaced-repetition/data.json:1]: with the new value, it should look like this on PC:
image

**Review by sethonne — APPROVED** [review]
